src/components/TableList/__snapshots__/TableList.test.js.snap

```diff
@@ -5,7 +5,87 @@ Array [
   <th
     role="columnheader"
   >
-    Name
```

Comment:
These snapshots aren't great. Let's plan to fix them during the styling/markup changes.